gitlab-runner-17.1.yaml

```diff
@@ -27,15 +30,22 @@ pipeline:
     with:
       deps: github.com/Azure/azure-sdk-for-go/sdk/azidentity@v1.6.0 github.com/hashicorp/go-retryablehttp@v0.7.7
 
+  - runs: |
+      mkdir -p "${{targets.destdir}}"/
```

Comment:
better to use contextdir wherever you need to

Reply:
Used contextdir instead